### DISCUSSION TITLE
Is it possible to set the CefSettings.IgnoreCertificateErrors Property in a config file or registry

### DISCUSSION THREAD

**Discussion by Jeff-W-Breadner** [cefsharp/CefSharp#3736] [General]
We've run into an issue where we need to set this flag. We see it can be set via C# code in the initialization method, but this would require us to push an updated binary out to a large existing client base. We are hoping that a configuration file, registry or some other method would allow us to change this setting without having to update the build.

**Comment by amaitland** [comment]
> We are hoping that a configuration file, registry

There is no built in support for either. 

> some other method would allow us to change this setting without having to update the build.

You can add --ignore-certificate-errors to the command 

```
myapp.exe --ignore-certificate-errors
```

**Reply by Jeff-W-Breadner** [reply]
Thanks for the quick response. I had found the command line parameters while investigating the issue, but was unaware that they were passed straight through from the client exe. Makes sense now that I know.

We appreciate the assistance

**Comment by Jeff-W-Breadner** [comment]
Related followup question. The reason we need to set this property is because an existing app that is using cefsharp (an older version 57.0.0) stopped working while using https against the Google Maps API. 

Is it the case that a certificate expired for that older cefsharp code ?  I was going to look into updating to the new version, but was curious as to why it just stopped working the other day.
